From e4afbecd79cc7e926c89c79bd9a399c9a8265319 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Wed, 23 Aug 2017 15:00:52 +0800
Subject: [PATCH 2546/5242] MLK-16475-02 arm64: dts: imx8qm/qxp: pull up uart
 pins in default

commit  c06e5ad8055e5fa35954cd29c60fd5e417b6c5b9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Pull up uart pins in default.

(BuildInfo: SCFW 3e70523d, IMX-MKIMAGE 0, ATF 0)

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |   20 ++++++++++----------
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |   16 ++++++++--------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |   12 ++++++------
 3 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index f535b40..cdd0fa1 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -253,26 +253,26 @@
 
 		pinctrl_lpuart0: lpuart0grp {
 			fsl,pins = <
-				SC_P_UART0_RX_DMA_UART0_RX		0x0600004c
-				SC_P_UART0_TX_DMA_UART0_TX		0x0600004c
-				SC_P_UART0_RTS_B_DMA_UART0_RTS_B	0x0600004c
-				SC_P_UART0_CTS_B_DMA_UART0_CTS_B	0x0600004c
+				SC_P_UART0_RX_DMA_UART0_RX		0x06000020
+				SC_P_UART0_TX_DMA_UART0_TX		0x06000020
+				SC_P_UART0_RTS_B_DMA_UART0_RTS_B	0x06000020
+				SC_P_UART0_CTS_B_DMA_UART0_CTS_B	0x06000020
 			>;
 		};
 
 		pinctrl_lpuart1: lpuart1grp {
 			fsl,pins = <
-				SC_P_UART1_RX_DMA_UART1_RX		0x0600004c
-				SC_P_UART1_TX_DMA_UART1_TX		0x0600004c
-				SC_P_UART1_CTS_B_DMA_UART1_CTS_B	0x0600004c
-				SC_P_UART1_RTS_B_DMA_UART1_RTS_B	0x0600004c
+				SC_P_UART1_RX_DMA_UART1_RX		0x06000020
+				SC_P_UART1_TX_DMA_UART1_TX		0x06000020
+				SC_P_UART1_CTS_B_DMA_UART1_CTS_B	0x06000020
+				SC_P_UART1_RTS_B_DMA_UART1_RTS_B	0x06000020
 			>;
 		};
 
 		pinctrl_lpuart3: lpuart3grp {
 			fsl,pins = <
-				SC_P_M41_GPIO0_00_DMA_UART3_RX		0x0600004c
-				SC_P_M41_GPIO0_01_DMA_UART3_TX		0x0600004c
+				SC_P_M41_GPIO0_00_DMA_UART3_RX		0x06000020
+				SC_P_M41_GPIO0_01_DMA_UART3_TX		0x06000020
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 67c13d6..0b0a3b4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -242,24 +242,24 @@
 
 		pinctrl_lpuart0: lpuart0grp {
 			fsl,pins = <
-				SC_P_UART0_RX_ADMA_UART0_RX	0x0600004c
-				SC_P_UART0_TX_ADMA_UART0_TX	0x0600004c
+				SC_P_UART0_RX_ADMA_UART0_RX	0x0600002c
+				SC_P_UART0_TX_ADMA_UART0_TX	0x0600002c
 			>;
 		};
 
 		pinctrl_lpuart1: lpuart1grp {
 			fsl,pins = <
-				SC_P_UART1_TX_ADMA_UART1_TX		0x0600004c
-				SC_P_UART1_RX_ADMA_UART1_RX		0x0600004c
-				SC_P_UART1_RTS_B_ADMA_UART1_RTS_B	0x0600004c
-				SC_P_UART1_CTS_B_ADMA_UART1_CTS_B	0x0600004c
+				SC_P_UART1_TX_ADMA_UART1_TX		0x0600002c
+				SC_P_UART1_RX_ADMA_UART1_RX		0x0600002c
+				SC_P_UART1_RTS_B_ADMA_UART1_RTS_B	0x0600002c
+				SC_P_UART1_CTS_B_ADMA_UART1_CTS_B	0x0600002c
 			>;
 		};
 
 		pinctrl_lpuart3: lpuart3grp {
 			fsl,pins = <
-				SC_P_FLEXCAN2_RX_ADMA_UART3_RX	0x0600004c
-				SC_P_FLEXCAN2_TX_ADMA_UART3_TX	0x0600004c
+				SC_P_FLEXCAN2_RX_ADMA_UART3_RX	0x0600002c
+				SC_P_FLEXCAN2_TX_ADMA_UART3_TX	0x0600002c
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 88edfa5..070c854 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -140,17 +140,17 @@
 
 		pinctrl_lpuart0: lpuart0grp {
 			fsl,pins = <
-				SC_P_UART0_RX_ADMA_UART0_RX	0x0600004c
-				SC_P_UART0_TX_ADMA_UART0_TX	0x0600004c
+				SC_P_UART0_RX_ADMA_UART0_RX	0x0600002c
+				SC_P_UART0_TX_ADMA_UART0_TX	0x0600002c
 			>;
 		};
 
 		pinctrl_lpuart1: lpuart1grp {
 			fsl,pins = <
-				SC_P_UART1_TX_ADMA_UART1_TX		0x0600004c
-				SC_P_UART1_RX_ADMA_UART1_RX		0x0600004c
-				SC_P_UART1_RTS_B_ADMA_UART1_RTS_B	0x0600004c
-				SC_P_UART1_CTS_B_ADMA_UART1_CTS_B	0x0600004c
+				SC_P_UART1_TX_ADMA_UART1_TX		0x0600002c
+				SC_P_UART1_RX_ADMA_UART1_RX		0x0600002c
+				SC_P_UART1_RTS_B_ADMA_UART1_RTS_B	0x0600002c
+				SC_P_UART1_CTS_B_ADMA_UART1_CTS_B	0x0600002c
 			>;
 		};
 
-- 
1.7.9.5

